docs/design/supply/network-import-tned.md

```diff
@@ -1,3 +1,113 @@
 # Network Import from TNED
 
-Details of importing and processing the ETL network.
\ No newline at end of file
+This section describes the procedure by which the ABM3 model system imports (into Emme) network (highway and transit) files along with a general description of the different network files.
+
+## Network Files
+
+The ABM3 model system has been configured to be compatible with SANDAG's Transportation Network Editing Database (TNED) system, which is utilized to edit, maintain and generate transportation networks. The TNED network files, generated via an ETL (i.e., Extract, Tranform, Load) procedure, serve as inputs to the ABM3 model system's import network procedure and are produced in text file, shapefile, geodatabase table and geodatabase feature class geodatabase formats. There are, additionally, some non-TNED input network files which are manually maintained.
+
+The following are the required network files used during the Emme import network procedure:
+
+<table>
```

Comment:
@cundo92 please use markdown for tables and not html. You can see some examples here - https://github.com/SANDAG/ABM/blob/ABM3_develop/docs/application/flexible-fleets.md

Reply:
@bhargavasana done. Converted HTML table to Markdown.